### DISCUSSION TITLE
Xresources string for lockscreen image non-tiled?

### DISCUSSION THREAD

**Discussion by nrvale0** [regolith-linux/regolith-desktop#1041] [Q&A] [answered]
Howdy. Not finding it in the docs nor in the code...

Is there an Xresources string I can set to set the lockscreen wallpaper but have it be centered or scaled but **not** tiled?

thx!

**Comment by w18g** [comment, marked as answer]
I did this following command: 
regolith-diagnostic | grep wallpaper

Seems like the Xresource is regolith.lockscreen.wallpaper.options, which mine is set at zoom.

> To change the image treatment (zoom, scale, etc.) of the wallpaper image, set regolith.wallpaper.options. Options are: none, wallpaper, centered, scaled, stretched, zoom, spanned

**Reply by nrvale0** [reply]
Oh! Very clever! ty!
